syntax = "proto3";

package text_generator;

service TextGenerator {
  rpc SendPrompt (TextPromptRequest) returns (TextPromptResponse) {}
}

message TextPromptRequest {
  string prompt = 1;
}

message TextPromptResponse {
  string message = 1;
}